required/tools/array.dtx

```diff
@@ -3517,7 +3517,46 @@ Bug reports can be opened (category \texttt{#1}) at\\%
 %  \end{macro}
 %
 %
+% \subsection{Handling \cs{cline}}
 %
+% In the past \pkg{array} did not have to concern itself with
+% \cs{cline} but simply used the definition already provided in the
+% kernel. However, for tagged PDF output this definition is
+% insufficient, because it causes incorrect row counting and the rules
+% it generates would need to be marked as artifacts.
+% We therefore update it  here.  
+%
+%
+%  \begin{macro}{\@cline}
+%    Tagging support for \cs{cline}
+%  \changes{v2.6e}{2024/07/13}{Support for tagging \cs{cline} (tagging/134)}
+%    \begin{macrocode}
+\ExplSyntaxOn
+\def\@cline#1-#2\@nil{
+  \omit
+  \@multicnt#1
+  \advance\@multispan\m@ne
+  \ifnum\@multicnt=\@ne\@firstofone{&\omit}\fi
+  \@multicnt#2
+  \advance\@multicnt-#1
+  \advance\@multispan\@ne
+%    \end{macrocode}
+%    The rule needs artifact tagging in tagged PDF.
+%    \begin{macrocode}
+  \UseSocket{tagsupport/tbl/leaders/begin}
+  \leaders\hrule\@height\arrayrulewidth\hfill
+  \UseSocket{tagsupport/tbl/leaders/end}
+%    \end{macrocode}
```

Comment:
Shouldn't that be \UseTaggingSocket{tbl/leaders/begin} etc?

Reply:
yes it should